src/targets/node/nodeLauncher.ts

```diff
@@ -66,7 +66,11 @@ export class NodeLauncher extends NodeLauncherBase<INodeLaunchConfiguration> {
     let config: INodeLaunchConfiguration | undefined;
     if (params.type === DebugType.Node && params.request === 'launch') {
       config = { ...params };
-    } else if (params.type === DebugType.Chrome && params.server && 'program' in params.server) {
+    } else if (
+      (params.type === DebugType.Chrome || params.type === DebugType.Edge) &&
+      params.server &&
+      'program' in params.server
+    ) {
```

Comment:
We can actually use `in` and TS will narrow and type check for us 🙂 

```suggestion
    } else if ('server' in params && params.server && 'program' in params.server) {
```

Reply:
Updated -- thank you!!